corpan/dja/cor/packs/service.py

```diff
@@ -0,0 +1,134 @@
+from typing import List, Tuple, Dict, Optional
+from django.db import transaction
+from corpora_ai.provider_loader import load_llm_provider
+
+from cor.models import Language, Entry, Pack, PackEntry, Translation
+from cor.utils.split import split_into_utterances
+from cor.utils.llm_source import translate_source_to_english_batch
+from cor.utils.llm import translate_entry_batch  # EN -> target
+
+DEFAULT_BATCH_SIZE = 40
+
+
+def create_pack_from_text(
```

Comment:
this looks like it could be the start of something special - but, I didn't mutate the DB yet with any packs. I think we make a version that just yields all of the entries/translations instead of this one that is coupled to Django. It would be nice to iterate through the results in a vanilla way and we could feed them to Django or parquet or anything ... we still need to come up with an elegant way to add packs that don't ship with the application binary in the sqlite ...

still, we can use this in a next release to add some key packs to the core corpus - like the 300 wise sayings for each language Pack ...

I want to start going nuts with Packs though ASAP - but, really we need to solve the problem of how to download them and graft them into the UI flow.